tests/posix/common/src/pthread.c

```diff
@@ -539,6 +539,35 @@ ZTEST(pthread, test_pthread_testcancel)
 	zassert_false(testcancel_failed);
 }
 
+static void *test_pthread_setschedprio_fn(void *arg)
+{
+	pthread_t self;
+	int prio = K_LOWEST_APPLICATION_THREAD_PRIO;
```

Comment:
This needs to be converted to a posix priority.

Generally, `0` will always be a valid posix priority   in case that is easier.